CK3ToEU4Tests/MapperTests/CultureMapper/CultureMappingTests.cpp

```diff
@@ -0,0 +1,217 @@
+#include "../../CK3toEU4/Source/Mappers/CultureMapper/CultureMappingRule.h"
+#include "gtest/gtest.h"
+#include <sstream>
+
+TEST(Mappers_CultureMappingTests, eu4CultureDefaultsToBlank)
```

Comment:
Also, ideally, you should devise tests that exercise the intended behavior of the class, not check its internal state (especially when that involves opening a bunch of getters you don't really care about in the end).